ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -388,6 +400,10 @@ public void RemoveRootView(int rootViewTag)
             DropView(rootView);
             _rootTags.Remove(rootViewTag);
 
+#if WINDOWS_UWP
+            AccessibilityHelper.OnRootViewRemoved(rootView as UIElement);
+#endif
```

Comment:
Do we want to call this with null?

Reply:
Especially since other calls to AccessibilityHelper use the `is` check.

---
In reply to: [177184582](https://github.com/Microsoft/react-native-windows/pull/1733#discussion_r177184582) [](ancestors = 177184582)

Reply:
K, I'll do that for consistency.

---
In reply to: [177185092](https://github.com/Microsoft/react-native-windows/pull/1733#discussion_r177185092) [](ancestors = 177185092,177184582)